code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1151,6 +1151,8 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 					if(85 to INFINITY)
 						damageoverlay.overlays += brutefireloss_overlays["6"]//image("icon" = 'icons/mob/screen1_full.dmi', "icon_state" = "brutedamageoverlay6")
 				//damageoverlay.overlays += I
+
+		see_invisible = SEE_INVISIBLE_LIVING

```

Comment:
The reason that's there is because anything that changed your invisibility flags fucked shit up, there's not code (except that line I just added) to reset them, which causes problems, as you could imagine.